react/src/js/components/Consonant/Container/Container.jsx

```diff
@@ -429,17 +429,19 @@ const Container = (props) => {
      * @param {Number} id - the id of an individual filter item
      * @returns {Void} - an updated state
      */
-    const clearFilterItem = (id) => {
+    const clearFilterItem = (id, group = '') => {
         setFilters((prevFilters) => {
             const filterClearedState = getFilterItemClearedState(id, prevFilters);
             return filterClearedState;
         });
 
         const urlParams = new URLSearchParams(window.location.search);
         clearUrlState();
+        // actually clear the url state
         urlParams.forEach((value, key) => {
             const chFilter = key.toLowerCase().replace('ch_', '').replace(' ', '-');
-            if (key.indexOf(filterGroupPrefix) !== 0 || !id.includes(chFilter)) {
+            if (key.indexOf(filterGroupPrefix) !== 0 || !id.toLowerCase().includes(chFilter) ||
+                !group.toLowerCase().includes(chFilter)) {
```

Comment:
Since there are more than two conditions here I would go with the following for better readability
```
if (key.indexOf(filterGroupPrefix) !== 0 
    || !id.toLowerCase().includes(chFilter)
    || !group.toLowerCase().includes(chFilter)) {
```